XCUITests/PrivateBrowsingTest.swift

```diff
@@ -97,30 +95,36 @@ class PrivateBrowsingTest: BaseTestCase {
         while settingsTableView.staticTexts["Close Private Tabs"].exists == false {
             settingsTableView.swipeUp()
         }
-
+        
         let closePrivateTabsSwitch = settingsTableView.switches["Close Private Tabs, When Leaving Private Browsing"]
-
+        
         XCTAssertFalse(closePrivateTabsSwitch.isSelected)
-
+        
         //  Open a Private tab
         navigator.goto(PrivateTabTray)
         navigator.openURL(urlString: url1)
         navigator.nowAt(PrivateBrowserTab)
         navigator.goto(PrivateTabTray)
-
+        
         // Go back to regular browser
         navigator.goto(TabTray)
-
+        
         // Go back to private browsing and check that the tab has not been closed
         navigator.goto(PrivateTabTray)
         waitforExistence(app.collectionViews.cells[url1Label])
         let numPrivTabs = app.collectionViews.cells.count
         XCTAssertEqual(numPrivTabs, 1, "The number of tabs is not correct, the private tab should not have been closed")
-
+        
+        app.collectionViews.cells[url1Label].tap()
         // Now the enable the Close Private Tabs when closing the Private Browsing Button
-        navigator.goto(SettingsScreen)
+        //        navigator.goto(SettingsScreen)
+        app/*@START_MENU_TOKEN@*/.buttons["TabToolbar.menuButton"]/*[[".buttons[\"Menu\"]",".buttons[\"TabToolbar.menuButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        let settingsmenuitemCell = app.tables.cells["Settings"]
```

Comment:
hmm, navigator.goto(SettingsScreen) should work- could you tell me what problem you run into? I'll fix FxScreenGraph